ReactiveXComponent/Parser/StateMachineInfo.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ReactiveXComponent.Parser
+{
+    public class StateMachineInfo
+    {
```

Comment:
Create a constructor with 2 parameters and remove the public setters

Reply:
done